uw-frame-components/portal/features/partials/announcement.html

```diff
@@ -21,7 +21,7 @@
     <md-menu-item layout="row" layout-align="space-between center">
       <span>Announcements</span>
       <a href="features" ng-click="markAnnouncementsSeen(true)" aria-label="see all announcements" class="link__see-all">
```

Comment:
Yes, it marks the announcements seen and takes the user to the "New features" page.